ta/pkcs11/src/persistent_token.c

```diff
@@ -13,31 +13,103 @@
 #include "pkcs11_token.h"
 #include "pkcs11_helpers.h"
 
+/*
+ * Token persistent objects
+ *
+ * The persistent objects are each identified by a UUID.
+ * The persistent object database stores the list of the UUIDs registered. For
+ * each it is expected that a file of ID "UUID" is store in the OP-TEE secure
+ * storage.
+ */
+static TEE_Result get_db_file_name(struct ck_token *token,
+				   char *name, size_t size)
+{
+	int n = snprintf(name, size, "token.db.%1d", get_token_id(token));
+
+	if (n < 0 || (size_t)n >= size)
+		return TEE_ERROR_SECURITY;
+	else
+		return TEE_SUCCESS;
+}
+
+static TEE_Result open_db_file(struct ck_token *token,
+			       TEE_ObjectHandle *out_hdl)
+{
+	char file[32] = { };
+	TEE_Result res = TEE_ERROR_GENERIC;
+
+	res = get_db_file_name(token, file, sizeof(file));
+	if (res)
+		return res;
+
+	return TEE_OpenPersistentObject(TEE_STORAGE_PRIVATE, file, sizeof(file),
+					TEE_DATA_FLAG_ACCESS_READ |
+					TEE_DATA_FLAG_ACCESS_WRITE,
+					out_hdl);
+}
+
+static void close_db_file(TEE_ObjectHandle hdl)
+{
+	TEE_CloseObject(hdl);
```

Comment:
Do you expect to ever do anything more in this function? If not, then I think we could just as well skip it.

Reply:
no more than that. Ok, i'll call `TEE_CloseObject()` straight.